examples/nemo/scripts/KGOs/namelist_cfg_test

```diff
@@ -7,16 +7,16 @@
 !-----------------------------------------------------------------------
    cn_exp      =   'BENCH' !  experience name
    nn_it000    =       1   !  first time step
-   nn_itend    =      10   !  last time step 
+   nn_itend    =    100   !  last time step 
    nn_stock    =      -1   !  frequency of creation of a restart file (modulo referenced to 1)
    nn_write    =      -1   !  frequency of write in the output file   (modulo referenced to nn_it000)
 /
 !-----------------------------------------------------------------------
 &namusr_def    !   User defined :   BENCH configuration: Flat bottom, beta-plane
 !-----------------------------------------------------------------------
-   nn_isize   =   360      ! number of point in i-direction of global(local) domain if >0 (<0)  
```

Comment:
Since the size of BENCH is easy to adjust we could, in future, have a bigger domain for the GPU tests.

Reply:
I agree, I have not look at timings at all here. I may start choosing something more sensible and recording them with #2697